tests/driver.sh

```diff
@@ -132,7 +156,7 @@ items 30 "int i; int acc; i = 0; acc = 0; do { i = i + 1; if (i - 1 < 5) continu
 items 26 "int acc; acc = 0; int i; for (i = 0; i < 100; i++) { if (i < 5) continue; if (i == 9) break; acc = acc + i; } return acc;"
 
 # functions
-try_ 55 << EOF
+try_ 55 "" << EOF
```

Comment:
Can you avoid `""` here?

Reply:
I think there are two cases to call `try_` function
1. compare both exit code and output
2. compare exit code only
Thus, there are two solutions.
1. Another argument check in `try_`
2. write another testing function like `try_`